app/assets/javascripts/models/page/project_page.js.coffee

```diff
@@ -56,6 +60,43 @@ ns.ProjectPage = do (ko,
       setTimeout((=>
         @_loadFromUrl()
         $(window).on 'edsc.pagechange', @_loadFromUrl), 0)
+   
+    showType: =>
+      type = ""
+      if @query.serialize().bounding_box then type = "Rectangle"
+      if @query.serialize().polygon then type = "Polygon"
+      if @query.serialize().point then type = "Point"
+      type
+  
+    hasType: =>
+      @query.serialize().bounding_box || @query.serialize().polygon || @query.serialize().point
+      
+    showTemporal: =>
```

Comment:
@jboukes can you remove all of this temporal code as well. This code should be introduced with the appropriate label and stuff as well. And secondary to that, this seems like a lot of code that is deconstructing and reconstructing date formats. Surely there's a better way to reformat a date and show only the bits we want? Is there something else going on here that I'm not seeing? Either way... we should take this conversation over to the appropriate branch/ticket.

Reply:
@mreese84 Back when that code was written, we'd talked about just commenting it out until we get to the temporal ticket (which, uncalled code kind of is 'commented out' since it's not executed).

However - not a prob.  :)  Will update in 5, 4, 3, 2,...

Reply:
Done!

As far as the other question, the code is a bit customized to the date format in order to get it closer to what was shown in the mock ups.  The way it is written is this:

If the year is the same between the dates, then show it as this:
Jan 12 - Mar 5, 2017

If the year *and* the month is the same between the dates, then show it as this:
Jan 12-15, 2017

Otherwise, show the dates in the more standard:
Jan 12, 2017 - Jan 12, 2018

We can talk more about that when we get to that ticket.